pkg/cloud/packet/client.go

```diff
@@ -45,8 +47,6 @@ const (
 )
 
 var (
-	clientName                     = "CAPP-v1beta1"
```

Comment:
I think we're still better off keeping clientName and clientUAFormat as `vars` so they can be modified at build time.
`const`s can not be modified at build time.